### PR TITLE
♻️ Exposes to front-end whether payment plugin is enabled (⚠️ ops)

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Informs the front-end whether the payments plugin (under development) is enabled or disabled.

- 🔨 If payment plugin is disabled, it is listed in the static-front-end-data config reponse . 
For instance with `WEBSERVER_PAYMENTS=null` the front-end will receive
image
therefore all entrypoints for `payments` are not added to the routes and any call will result in a **404 (not found)**. If `WEBSERVER_PAYMENTS` is enabled the it does not appear in the list
image
- ♻️ some cleanup
## Related issue/s

- Related to https://github.com/ITISFoundation/osparc-simcore/issues/4657

## How to test

in-place

## DevOps ⚠️ 

- production must set `WEBSERVER_PAYMENTS=null`
- both staging and master can test payments by setting  `WEBSERVER_PAYMENTS={}` and letting the granulated envvars take precedence ie. `PAYMENTS_*` env-vars

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
looking good :+1:

**Review by GitHK — APPROVED** [review]
👍

**Review by odeimaiz — APPROVED** [review]
Merci

**Comment by mrnicegyu11** [comment]
re: DevOps changes: Ack!

File: services/web/server/src/simcore_service_webserver/payments/settings.py [hunk 1]
```diff
@@ -68,9 +67,9 @@ def base_url(self) -> str:
 
     @validator("PAYMENTS_FAKE_COMPLETION")
     @classmethod
-    def check_dev_feature_enabled(cls, v):
-        if v and not os.environ.get("WEBSERVER_DEV_FEATURES_ENABLED", False):
-            msg = "PAYMENTS_FAKE_COMPLETION only allowed when WEBSERVER_DEV_FEATURES_ENABLED=1"
+    def _payments_cannot_be_faken_in_production(cls, v):
+        if v is True and "production" in os.environ.get("SWARM_STACK_NAME", ""):
```
[the remark this answers is not in the document]
Comment (pcrespov) [reply services/web/server/src/simcore_service_webserver/payments/settings.py:71]: @GitHK  I agree.

**Pull request merged**